From 9c3718497935093e54317ecfa9f3e9114ea9c07c Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 02 Aug 2022 16:55:10 +0300
Subject: [PATCH] [SW-0] erom: recovery: remove unused callback

spi_flash_prot_late() callback is only used on Goya Preboot and in eROM
resolves to nothing.
The callback is unneeded in eROM - remove it.

Change-Id: I26b914905f4bba8bab3d865bc6e1927cd47e3ca9
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/ext_rom/erom_recovery.c b/app/ext_rom/erom_recovery.c
index 16aa192..67cb4a9 100644
--- a/app/ext_rom/erom_recovery.c
+++ b/app/ext_rom/erom_recovery.c
@@ -112,7 +112,6 @@
 	preboot_cond_arc0_reset();
 
 	spi_flash_init();
-	spi_flash_prot_late();
 	spi_flash_prot_status();
 
 	preboot_sp_sram_init();
